### ISSUE TITLE
username link on issue comments

### ISSUE THREAD

**Issue by kaos** [mattermost/mattermost-integration-gitlab#11]
seems to always point at gitlab.com/u/<username>...

**Comment by NotSqrt** [comment]
Looks like the user url is not in the sent data (https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/web_hooks/web_hooks.md), it will have to be built by extracting the root url from one of the urls sent by the webhook.

**Comment by kaos** [comment]
Yep, that would be the fix :)

**Comment by jwilander** [comment]
Yes, I can work on a fix for this in a day or say unless someone else wants to submit a PR for it :)

**Comment by kaos** [comment]
I wouldn't mind submitting a PR, but haven't got the time right now...

**Comment by NotSqrt** [comment]
I might look into it, and also on #8 , since I'm the one that reported it :smiley:

**Comment by NotSqrt** [comment]
Done in my repository.
